playground/backend/internal/code_processing/code_processing.go

```diff
@@ -110,20 +102,9 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	// run
 	logger.Infof("%s: Run() ...\n", pipelineId)
 	runCmd := executor.Run(ctxWithTimeout)
-
 	var runError bytes.Buffer
-	runOutput := &streaming.RunOutputWriter{Ctx: ctxWithTimeout, CacheService: cacheService, PipelineId: pipelineId}
-	runCmd.Stderr = &runError
-	runCmd.Stdout = runOutput
-	go func(cmd *exec.Cmd, successChannel chan bool, errChannel chan error, dataChannel chan interface{}) {
-		err := cmd.Run()
-		if err != nil {
-			errChannel <- err
-			successChannel <- false
-		} else {
-			successChannel <- true
-		}
-	}(runCmd, successChannel, errorChannel, dataChannel)
+	runOutput := streaming.RunOutputWriter{Ctx: ctxWithTimeout, CacheService: cacheService, PipelineId: pipelineId}
```

Comment:
some users may want to read stderr in the frontend as well as stdout, so maybe it makes sense to write that to the cache as well.
if I remember correctly, LOG data is weitten to stderr, while only System.out.println-type data will be sent to stdout